db/schema.rb

```diff
@@ -228,7 +229,6 @@
     t.datetime "last_sign_in_at"
     t.index ["confirmation_token"], name: "index_email_addresses_on_confirmation_token", unique: true
     t.index ["email_fingerprint", "user_id"], name: "index_email_addresses_on_email_fingerprint_and_user_id", unique: true
-    t.index ["email_fingerprint"], name: "index_email_addresses_on_all_email_fingerprints"
```

Comment:
this seems like some noise from another #4148, is it just merge stuff? Can we clean it up?

Reply:
> this seems like some noise from another #4148, is it just merge stuff? Can we clean it up?

Resolved conflicts in rebased b5f1334ea68533771810bbf9b83022da279d3482.